core/scss/components/site-search/_site-search.scss

```diff
@@ -37,18 +37,13 @@
   @include padding(0);
   @include margin(0 0 0 null);
 
-  background-color: transparent;
+  background: url("#{$image-path}/icon-search.svg") no-repeat 0 0;
   opacity: .6;
   position: absolute;
   top: 11px;
   right: 12px;
-  width: auto;
-
-  img {
-    display: inline-block;
-    max-width: 24px;
-    max-height: 24px;
-  }
+  width: 2.8rem;
+  height: 2.8rem;
```

Comment:
The units for top & right vs width & height seem switched to me. The units for width and height are relative (to the document's font-size), but the image has a fixed width (24) and height).

Conversely, the units for top and right are fixed (in px), but the size of the input box (at least the width) might conceivably change. It appears the height is fixed at 40px, which I now question: if low-vision users set their base font size very large, will they be able to see what they type in the search field?

Anyway, are we sure we're using the right units for top / right vs. width / height?

Reply:
Let me check - it could be that svg was finicky. In some cases they appear ok with px values but jagged with em/rem values. It's also an earlier PR so it's possible that I could improve some of these things.